node/deepgram_topic_detection/src/index.js

```diff
@@ -0,0 +1,33 @@
+const { Deepgram } = require("@deepgram/sdk");
+
+module.exports = async (req, res) => {
+  const apiKey = req.variables["DEEPGRAM_API_KEY"];
+  const fileUrl = req.payload["fileUrl"];
+
+  if (!apiKey) {
+    throw new Error("API key is required");
+  }
+
+  if (!fileUrl) {
+    throw new Error("wav file url is required");
```

Comment:
Instead of throwing an error, make sure to call `res.json()` with `success` = `false` to match the issue requirements.

Reply:
Got it.